packages/action-group/test/action-group.test.ts

```diff
@@ -349,16 +349,18 @@ describe('ActionGroup', () => {
                 },
             ],
         });
-        await opened;
         await elementUpdated(el);
+        await opened;
 
         expect(actionMenu).to.equal(document.activeElement);
         const closed = oneEvent(el.children[3] as ActionMenu, 'sp-closed');
 
         // use keyboard to navigate to the second menu item and select it
         await sendKeys({ press: 'ArrowDown' });
+        await elementUpdated(el);
```

Comment:
Nice hardening here to wait for the element to finish updates before moving on to the next validation step.